lib/rules/stylelint-disable-no-empty/index.js

```diff
@@ -0,0 +1,54 @@
+"use strict"
+
+const report = require("../../utils/report")
+const ruleMessages = require("../../utils/ruleMessages")
+const validateOptions = require("../../utils/validateOptions")
+
+const ruleName = "stylelint-disable-no-empty"
+
+const messages = ruleMessages(ruleName, {
+  rejected: "Unexpected empty `stylelint-disable` comment",
+})
+
+const stylelintDisableCommand = "stylelint-disable"
+
+const rule = (actual) => {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, { actual })
+    if (!validOptions) {
+      return
+    }
+
+    root.walkComments((comment) => {
+      if (!isDisableCommand(comment.text)) {
+        return
+      }
+
+      const specificRules = comment.text.split(" ", 2).slice(1)
```

Comment:
Yeah it seems like it does get trimmed, but maybe it should be more precise than this.

For example doing `/* stylelint-disable , */` would actually pass the rule, but I don't think I can validate if the rule itself is correct, or maybe it should be separate to this rule. Then you have `/* stylelint-disable  block-no-empty */` with two spaces in the middle causing that part of the code to think there are two rules in that line. I might tweak that to trim spaces around the list of rules being disabled.

Regardless I've added a few test cases that account for weird spacing anyway.